include/libopencm3/stm32/l0/rcc.h

```diff
@@ -490,55 +493,65 @@ enum rcc_periph_clken {
 	RCC_GPIOB = _REG_BIT(0x2c, 1),
 	RCC_GPIOC = _REG_BIT(0x2c, 2),
 	RCC_GPIOD = _REG_BIT(0x2c, 3),
+	RCC_GPIOE = _REG_BIT(0x2c, 4),
 	RCC_GPIOH = _REG_BIT(0x2c, 7),
 
 	/* AHB peripherals */
 	RCC_DMA = _REG_BIT(0x30, 0),
 	RCC_MIF = _REG_BIT(0x30, 8),
 	RCC_CRC = _REG_BIT(0x30, 12),
-	RCC_TOUCH = _REG_BIT(0x30, 16),
+	RCC_TOUCH = _REG_BIT(0x30, 16), /* Compatibility */
+	RCC_TSC = _REG_BIT(0x30, 16),
```

Comment:
Relooking at it, there was no TOUCH used anywhere else, so this is just old names from an obsolete version of the ref man, and l0 is still super new, so I don't think we should feel at all bad about compatibility at this point.  I would say just drop the _TOUCH entry, and say, "ST! you did a bad job with your early ref mans!"

RCC_TOUCH is currently only in l0, and only from teh rev1/rev2 of the refman.  RCC_TSC is already existing for F3/F0/L4 and is the ref man style for l0 now.  Let's just kill _TOUCH.  (Same argument for _FW just below here, and the earlier define for RCC_AHBSMENR_TOUCHSMEN.  Again though, we can probably just drop these in a standlone commit with that argument in it.